app/src/main/java/currencyConverter/codes/CodeUtils.java

```diff
@@ -0,0 +1,16 @@
+package currencyConverter.codes;
+
+import java.util.List;
+
+import currencyConverter.codes.CountryCode;
+import currencyConverter.codes.CurrencyCode;
+
+public class CodeUtils {
+
+    public static CurrencyCode getCurrencyFromCountry(CountryCode countryCode) {
+        List<com.neovisionaries.i18n.CurrencyCode> currencyCodeList =
+            com.neovisionaries.i18n.CurrencyCode.getByCountryIgnoreCase( countryCode.toStringISO() );
+
+        return new CurrencyCode( currencyCodeList.get(0).getName() );

```

Comment:
I think a good idea is to check if we have at least one item in the list. Otherwise we will get a NullPointerException